docs/test/action-groups.md

```diff
@@ -180,6 +180,34 @@ MFTF resolves `{{myCustomEntity.field1}}` the same as it would in a `selector` o
 </actionGroup>
 ```
 
+## Returning a value
+
+Action groups can return a value using `<return>` action.
+
+```xml
```

Comment:
The example below and the example in actions.md seem to be duplicate. I would keep one and make a link to the other. But maybe check with @dobooth

Reply:
fixed.